torch/distributed/fsdp/sharded_grad_scaler.py

```diff
@@ -83,6 +83,7 @@ class ShardedGradScaler(GradScaler):
 
     def __init__(
         self,
+        device: str = "cuda",
```

Comment:
If this is CUDA only, then shouldn't it inherit from the GradScaler from torch.cuda.amp.grad_scaler?